0000-template.md

```diff
@@ -89,3 +90,8 @@ readers of your RFC with a fuller picture.
 - What related issues do you consider out of scope for this RFC that could be
   addressed in the future independently of the solution that comes out of this
   RFC?
+
+# Changelog
+[changelog]: #changelog
+
+- [10 Jan 2019] - v2 - a one-line summary of the changes in this version.
```

Comment:
Should ChangeLog be required to include a link to the previous versions so its easy to see how it evolved?